Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGToDBAnalyzer.cs

```diff
@@ -247,6 +243,16 @@ public override int Analyze()
         /// <inheritdoc/>
         public override void Dispose()
         {
+            Analysis.IgnoreResult(

```

Comment:
You should probably move this inside the RocksDbStore (i.e. when it is done and working in bulk load mode, issue a manual compression on Dispose). There's no scenario in which this isn't needed, and seems like easy to miss.